shared/js/ui/models/site.es6.js

```diff
@@ -191,6 +196,18 @@ Site.prototype = window.$.extend({},
       return count
     },
 
+    getMajorTrackerNetworksCount: function () {
+      // console.log('[model] getMajorTrackersCount()')
+      return Object.keys(this.tab.trackers).reduce((total, name) => {
+        let tempTracker = name.toLowerCase()
+        const majorTrackingNetworks = Object.keys(window.constants.majorTrackingNetworks)
+          .filter((t) => t.toLowerCase() === tempTracker)
+        // in case a major tracking network is in the list more than once somehow
+        total += majorTrackingNetworks.length ? 1 : 0
```

Comment:
I could have just added majorTrackingNetworks.length here, but in the spirit of defensive programming...